projects/1.21/assets/theobsidianboat/obsidianboat/lang/zh_cn.json

```
@@ -0,0 +1,6 @@
+{
+  "advancements.obsidianboat.root.description": "你真的觉得这样行吗？",
+  "advancements.obsidianboat.root.title": "什么曾让你的船沉没",
```

```suggestion
  "advancements.obsidianboat.root.title": "乱来可是会翻船的",
```
原文梗的是俗语`Whatever floats your boat`，意思是`Do whatever makes you happy or stimulates you.`